### ISSUE TITLE
ExplainableBoostingRegressor not supporting other link functions

### ISSUE THREAD

**Issue by sadsquirrel369** [jpmml/sklearn2pmml#422]
```
Exception in thread "main" org.jpmml.python.AttributeException: Attribute 'interpret.glassbox._ebm._ebm.ExplainableBoostingRegressor.link_' has an unsupported value 'log'. Supported values are ['identity']
        at org.jpmml.python.PythonObject.getEnum(PythonObject.java:275)
        at interpret.glassbox.ebm.ExplainableBoostingRegressor.getLink(ExplainableBoostingRegressor.java:77)
        at interpret.glassbox.ebm.ExplainableBoostingRegressor.encodeModel(ExplainableBoostingRegressor.java:42)
        at sklearn.Estimator.encode(Estimator.java:126)
        at sklearn2pmml.pipeline.PMMLPipeline.encodePMML(PMMLPipeline.java:121)
        at com.sklearn2pmml.Main.run(Main.java:80)
        at com.sklearn2pmml.Main.main(Main.java:65)
```

**Comment by vruusmann** [comment]
Support for EBMs is a very recent functionality, and I decided to get started with the most common link functions - `identity` for regression and `logit` for binary classification.

Adding support for more link functions would be trivial - it's a matter of setting the `RegressionModel@normalizationMethod` attribute.

Right now, you can "disable" an unsupported link function by setting it to `identity`:

```
from sklearn2pmml import sklearn2pmml

regressor = ExplainableBoostingRegressor()
regressor.fit(X, y)

# THIS
regressor.link_ = "identity"

sklearn2pmml(regressor, "EBM.pmml")
```

Then, open the resulting "EBM.pmml" file in text editor, and set the `RegressionModel@normalizationMethod` attribute to whatever value you need.
